frontend-react/src/network/api/Api.ts

```diff
@@ -1,45 +1,63 @@
-import { AxiosRequestConfig } from "axios";
+import { AxiosRequestConfig, AxiosRequestHeaders } from "axios";
 
-import {
-    getStoredOktaToken,
-    getStoredOrg,
-} from "../../components/GlobalContextProvider";
+/* Type alias for CRUD ops that I wish to allow */
+export type HTTPMethod = "POST" | "GET" | "PATCH" | "DELETE";
```

Comment:
I left it this way because it was out of scope for this ticket, and we can edit it as needed for our project; this was the default I'd put in the packaged version

Reply:
I get that, but looking at it seems like a mistake was made by leaving out the "PUT"?
Maybe add a comment saying it's a TODO item?

(Adding comments to clarify things that confused me when reading through the code will make me happy most of the time.)